### ISSUE TITLE
Failed to build release with rust stable

### ISSUE THREAD

**Issue by mqy** [shadowsocks/shadowsocks-rust#359]
Here is the error when run `cargo +1.48.0-x86_64 build release`:

```
error[E0554]: `#![feature]` may not be used on the stable release channel
 --> /Users/mqy/.cargo/registry/src/github.com-1ecc6299db9ec823/crypto2-0.1.1/src/lib.rs:2:1
  |
2 | #![feature(stdsimd, llvm_asm)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
While `cargo +nightly-2020-12-05 build --release` works.

**Comment by zonyitoo** [comment]
Yes. It doesn't support stable Rust. Waiting for `stdsimd` to be stablized.

**Comment by mqy** [comment]
Use toolchain file?

**Comment by zonyitoo** [comment]
Yes. Already added in another branch. Will be merged with tokio v1.0 related modifications.
